packages/components/src/styles/container/container.stories.ts

```diff
@@ -52,178 +49,112 @@ export const Default = {
 };
 
 /**
- * Use the 5 color variants for alternate container experiences.
+ * Offers 5 color variants for alternate experiences:
```

Comment:
Offers “color” variants for alternate experiences:

Reply:
_For consistency use also here:_

Use the variant “classes” for alternative appearances: